packages/core/src/routes/admin-user.test.ts

```diff
@@ -113,24 +117,8 @@ describe('adminUserRoutes', () => {
 
   it('POST /users should throw with invalid input params', async () => {
     const username = 'MJAtLogto';
-    const password = 'PASSWORD';
     const name = 'Micheal';
 
-    // Missing input
-    await expect(userRequest.post('/users').send({})).resolves.toHaveProperty('status', 400);
-    await expect(userRequest.post('/users').send({ username, password })).resolves.toHaveProperty(
-      'status',
-      400
-    );
-    await expect(userRequest.post('/users').send({ username, name })).resolves.toHaveProperty(
-      'status',
-      400
-    );
-    await expect(userRequest.post('/users').send({ password, name })).resolves.toHaveProperty(
-      'status',
-      400
-    );
-
```

Comment:
we may add test cases that ensure we can not call this API without giving `primaryEmail` and `username` at the same time.

Reply:
i think no need for this since it's mgmt API

Reply:
discussed offline